src/Histogram.chpl

```diff
@@ -61,7 +61,7 @@ module Histogram
         return hist;
     }
 
-    proc histogramGlobalAtomic(x: [?aD] ?etype, y: [aD] etype, xMin: etype, xMax: etype, yMin: etype, yMax: etype, numXBins: int, numYBins: int, xBinWidth: real, yBinWidth: real) throws {
+    proc histogramGlobalAtomic(x: [?aD] ?etype1, y: [aD] ?etype2, xMin: etype1, xMax: etype1, yMin: etype2, yMax: etype2, numXBins: int, numYBins: int, xBinWidth: real, yBinWidth: real) throws {
         const totNumBins = numXBins * numYBins;
         var hD = makeDistDom(totNumBins);
         var atomicHist: [hD] atomic int;
```

Comment:
all the other `atomicHist` were turned into `real`, should this one be too?

Reply:
Added this and confirmed I was not using the right build to check multidim